Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -277,6 +277,12 @@ public void ReceiveMessage(Message message)
         }
 #pragma warning restore CS4014
 
+        public void RemovePeer(BoundPeer peer)
+        {
+            _logger.Debug("Removing peer {Peer} from table.", peer);
+            _routing.RemovePeer(peer);
+        }
+
```

Comment:
If this method become `public` shouldn't `IProtocol` interface have this method too?